packages/aurora-feed/index.js

```diff
@@ -105,10 +110,22 @@ class Feed extends React.Component {
     });
   }
 
+  onDelete(uuid) {
+    deleteNote(uuid);
+    this.setState(prevState => {
+      prevState.shownNotes = removeNoteData(prevState.shownNotes, uuid);
+      prevState.allNotes = removeNoteData(prevState.allNotes, uuid);
+      return prevState;
+    });
+  }
+
   onSubmit(editorState) {
     // Add a card with a copy of the editor state
-    this.addCard(editorState);
-    save(editorState);
+    let uuid = save(editorState);
+    this.addCard({
```

Comment:
NIT: Not a big deal, but JS will auto interpret the names of variable if you want. 

Example:
```js
const Cheese = "brie";
const foo = { Cheese };

/**
foo = { 
  Cheese: "brie" 
}
**/
```

So here you can just do:
``` js 
this.addCard({  editorState, uuid });
```

Note that this assumes that the name of the variable you're passing in is also the name of the JSON object's attribute.

Reply:
I think it was giving me an error when I tried to do something like this

Reply:
Ah, then don't worry about it. It's not like a bad thing, just more to type :P